content/en/docs/concepts/overview/working-with-objects/annotations.md

```diff
@@ -69,6 +69,23 @@ If the prefix is omitted, the annotation Key is presumed to be private to the us
 
 The `kubernetes.io/` and `k8s.io/` prefixes are reserved for Kubernetes core components.
 
+For example, the sample Pod with the annotation `imageregistry: https://hub.docker.com/`
```

Comment:
This is not a complete sentence.

Reply:
Changed statement to make it more meanigful